### ISSUE TITLE
Bug: doesn't parse object syntax for css properties unless the object is defined directly in the JSX

### ISSUE THREAD

**Issue by alex-e-leon** [stylelint/postcss-css-in-js#205]
> Clearly describe the bug
I'm using emotion in my project with `object styles` syntax. Stylelint parses and lints the following just fine:
```
<span css={{fontSize: '10px`}} />
```
But does not parse the following:
```
const styles = {fontSize: '10px`};
<span css={styles} />
```
or if I'd like spread and combine styles
```
const styles = {fontSize: '10px`};
<span css={{...styles}} />
```

> What steps are needed to reproduce the bug?

See above.

> What did you expect to happen?

That stylelint would parse and lint styles declared in an object that are passed into a component.

> What actually happened (e.g. what warnings or errors did you get)?

When defining styles outside of the JSX component stylelint just ignores the css completely, and no errors are reported.
